docs/psoc6/quickref.rst

```diff
@@ -669,10 +669,12 @@ I2S bus
 
 See :ref:`machine.I2S <machine.I2S>`. 
 
-Supported sample rates are 8KHz, 16KHz, 32KHz, 48KHz, 22.05KHz, 44.1KHz.
+.. note::
+    The only difference with the reference API is the usage of the ``id`` in the constructor ``IS2()``. In the PSoC6 port the ``id`` value is ignored. 
+    Any integer value can be passed as ``id``.
+    New instances of the I2S class will be allocated as long as there are available slots and the chosen I2S pins aren't already allocated by other instance.
```

Comment:
One more point missing is that "The id will be allocated with a value that signifies the assigned I2S instance."

Reply:
See comment to @IFX-Anusha

Reply:
Now changed :)